src/handlers.js

```diff
@@ -121,7 +121,7 @@ const handlers = {
                 R.filter(({ name }) => name === 'username'),
                 R.map(({ selected_options }) => selected_options),
                 R.unnest,
-                R.find(() => true),
```

Comment:
How does the data look coming from the R.unnest step?

Reply:
Maybe this refactor should come with a unit test aswell

Reply:
It's an array of objects with a prop `value`, if I recall correctly. And it should only ever be one selected option and object.

I'll look into writing a test as well!

Reply:
Hmm, if there is only ever one option could R,take(1) be used?

Reply:
What i really wanted was a "first" operator but didn't find any in Randa. :shrug: `take` would still yield a (singleton) list, right?

Reply:
...and apparently, it was [`head`](https://ramdajs.com/docs/#head) I was looking for. :eyes: